datamade/just-spaces
# Templated surveys

## Overview

Closes #90. This PR creates a pipeline for storing and importing survey templates, then allowing users to choose one when creating a survey. It also creates two example templated surveys. Real templates will be created later by UCD on the staging site and incorporated in #195.

**Note:** As this stands, `python manage.py import_survey_templates` will have to be run manually on the staging server. It cannot be added to `after_install` without also adding a user, which I don't think we want to do automatically right now.

## Testing Instructions

* Run migrations
* Import survey templates: `python manage.py import_survey_templates`
* Start creating a new survey at `localhost:800/surveys/create/`. Make sure the `Survey templates` field contains some options.
* Create the survey! Do you see some templated questions already loaded?
